ivy.el

```diff
@@ -840,6 +840,10 @@ selection, non-nil otherwise."
       (let ((ivy--directory ivy--directory))
         (funcall ivy-read-action-function actions)))))
 
+(defcustom ivy-exit-keys-from-actions '("ESC" "M-o")
+  "A list of extra keys to exit ivy actions list."
+  :type '(list string))
```

Comment:
This should probably be `(repeat key-sequence)` instead; see [`(info "(elisp) Customization Types")`](https://www.gnu.org/software/emacs/manual/html_node/elisp/Customization-Types.html).

Reply:
@basil-conto Thank you for the information.
I just fixed it.
Thank you again!